### ISSUE TITLE
Need to find multiple columns for the same model for prediction

### ISSUE THREAD

**Issue by Sonu2396** [dotnet/machinelearning#6680]
I have been doing regression for using the taxi dataset and have got the solution worked fine, however only one column can be predictied as of now and i want to have multiple columns predicted by the system .there  is no provision for that as of now. What should be done exactly.

**Comment by torronen** [comment]
One easy way would be to make one model per column you want to predict. For example, one model to predict fareAmount, another to predict tipAmount etc. You could even try to put the predicted input from first model (fareAmount) as input for the second model (tipAmount), or maybe even some random values for fareAmount and see how predicted tipAmount would change.

**Comment by Sonu2396** [comment]
That would still not solve the problem as it will predict for a different value and has to be predicted together. Furhter rejoing all of these will take some time. Also LSTM's are needed in most of the cases.

**Comment by michaelgsharp** [comment]
This is not something that we currently support, and I don't believe we have it anytime soon in our backlog.

Does that sound accurate @luisquintanilla?

**Comment by luisquintanilla** [comment]
> This is not something that we currently support, and I don't believe we have it anytime soon in our backlog.
> 
> Does that sound accurate @luisquintanilla?

Correct. This is not something we're considering in the near term and @torronen suggestion would be the recommended path at the moment.
